tests/nlu/parsers/test_digits.py

```diff
@@ -0,0 +1,35 @@
+"""
+This module contains the tests for the digits parser
+"""
+from spokestack.nlu.parsers import digits
+
+
+def test_digits():
```

Comment:
Check the Android version's tests here; it'd be good to have some sad paths here (unparsable input beyond the empty string, situations where the `count` metadata is violated, etc). Also things that exercise normalization and maybe some mixed numeric/word input.

Reply:
On the metadata violation, I originally sliced the count to the specified length. Is the desired behavior here to not return anything if the metadata is violated?

Reply:
Yes; values that don't match the facets are discarded by the parser. I didn't realize this mismatch the first time through, but your slot parsing doesn't match the API of the other platforms, which ensure that the application has access to the:
* slot name
* parsed value
* raw value recognized in the utterance
In the case of a facet violation, the parsed value would be null, but the raw value would still be present.